Tools/CMake/AMReXSYCL.cmake

```diff
@@ -0,0 +1,76 @@
+#
+# This module defines the INTERFACE target SYCL and its alias AMReX::SYCL.
+# These targets provides build/link requirements for the SYCL language.
+# For the time being, only dpc++  is supported
+#
+
+# Provide a cache variable for the dpc++ root directory installation by probing
+# the compiler
+execute_process(COMMAND ${CMAKE_CXX_COMPILER} --version  OUTPUT_VARIABLE _tmp)
+string(REGEX MATCH "InstalledDir: (.*)" _tmp "${_tmp}")
```

Comment:
I think this block needs some simple exit code (success) verification. There are many SYCL compilers out there.